tests/drivers/charger/sbs_charger/testcase.yaml

```diff
@@ -0,0 +1,31 @@
+tests:
+  # section.subsection
+  drivers.sbs_charger.emulated:
+    tags: drivers charger
+    filter: >
+      dt_compat_enabled("sbs,sbs-charger") and
+      (CONFIG_QEMU_TARGET or CONFIG_BOARD_NATIVE_POSIX)
+    extra_args:
+      CONF_FILE="prj.conf;boards/emulated_board.conf"
+      DTC_OVERLAY_FILE="boards/emulated_board.overlay"
+    platform_exclude:
+      qemu_cortex_a53
+      qemu_cortex_a53_smp
+      qemu_kvm_arm64
+      xenvm
+      xenvm_gicv3
+      hifive_unmatched
```

Comment:
I had to add an exclusion for this risc-v platform to dodge a devicetree overlay error. I'm not really sure of the root cause, but all other platforms are passing using the same overlay.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/56666#discussion_r1170539146
Probably a similar story